protocol.html

```diff
@@ -559,6 +572,40 @@ <h3 property="schema:name">Resource Containment</h3>
+                      <p>Container descriptions are not limited to containment triples. To further support client navigation and application interaction, servers can include <a href="#authoritative-information">authoritative information</a> about contained resources as part of the container description.</p>
+
+                      <p><span about="" id="authoritative-contained-resource-data" rel="spec:requirement" resource="authoritative-contained-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> include the following information about contained resources as part of the container description, unless that information does not exist or is expensive to determine:</span></span></p>
+
+                      <dl>
+                        <dt><code>rdf:type</code></dt>
+                        <dd>A class whose URI formed by concatenating <code>http://www.w3.org/ns/iana/media-types/</code>, the IANA Media Type [<cite><a class="bibref" href="#bib-iana-media-types">IANA-MEDIA-TYPES</a></cite>] of the resource, and <code>#Resource</code>.</dd>
+                        <dt><code>stat:size</code></dt>
+                        <dd>A non-negative integer giving the size of the resource in bytes.</dd>
+                        <dt><code>dcterms:modified</code></dt>
+                        <dd>The time when the resource was last modified.</dd>
+                        <dt><code>stat:mtime</code></dt>
+                        <dd>The Unix time when the resource was last modified.</dd>
+                      </dl>
+
+                      <div class="note" id="authoritative-resource-data-logically-part-of-container-description" inlist="" rel="schema:hasPart" resource="#authoritative-resource-data-logically-part-of-container-description">
+                        <h5 property="schema:name"><span>Note</span>: Authoritative Resource Data Logically Part of Container Description</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Authoritative data about contained resources is logically part of the container. Servers are encouraged to consider omitting authoritative data about a contained resource when agent is unauthorized to read the contained resource.</p>
+                        </div>
+                      </div>
+
+                      <p><span about="" id="contained-resource-state-cascading" rel="spec:requirement" resource="#contained-resource-state-cascading"><span property="spec:statement">When the state of a contained resource changes, the <span rel="spec:requirementSubject" resource="spec:Server">servers</span> <span rel="spec:requirementLevel" resource="spec:MUST-NOT">MUST NOT</span> expose any authoritative information that has a cascading potential to its immediate container.</span></span></p>
```

Comment:
I do not (fully) understand.

Reply:
@csarven is just trying to accommodate my concern from https://github.com/solid/specification/issues/227#issuecomment-931202286 but he can't. It is trying to reconcile two mutually exclusive ideas. If certain properties like mtime is defined as the mtime of the resource, it will be cascading. There's no way to avoid that. So, NSS doesn't do that, it defines it based on the mtime of the filesystem inode POSIX. So, the exercise we have to go through is, if we don't want to do like POSIX and use an auxiliary resource, to define mtime in a way that is consistent with Web principles, but not in terms of modification time for a resource. I've shouted as loud as I possibly can about this problem...

Reply:
Acknowledging the issue raised by @kjetilk, but at a minimum it could a be bit more clear for the reader what the text is trying to accomplish.

Reply:
Removed https://github.com/solid/specification/pull/352/commits/f2c8c8b82e2c40a1371f73c60671dfc612b2cbbd in favour of https://github.com/solid/specification/pull/352/commits/4eaf3456c4a8d90a2fb9fd931a8135733914da91 and https://github.com/solid/specification/pull/352/commits/30eee02aafb2077ee48c7ddbe1c596554184f3b8